src/data/ff/ff.cpp

```diff
@@ -57,20 +56,20 @@ EnumOptions<Forcefield::ShortRangeType> Forcefield::shortRangeTypes()
 
 // Add new atom type with its own parameters
 void Forcefield::addAtomType(Elements::Element Z, int index, std::string_view name, std::string_view netaDefinition,
-                             std::string_view description, double q, double data0, double data1, double data2, double data3)
+                             std::string_view description, double q, std::vector<double> parameters)
 {
```

Comment:
```suggestion
                             std::string_view description, double q, std::vector<double> &parameters)
{
```

Pass by reference.  Granted, a copy ultimately gets made, but at least we won't make two.